docs/filling_tests/index.md

```diff
@@ -0,0 +1,7 @@
+# Filling Tests
+
+Execution of test cases against clients is a two-step process. First JSON test fixtures are generated from the Python test cases in this repository using `fill`. Then the resulting JSON fixtures can be executed against a client. The process of generating fixtures is often referred to as "filling" the tests.
+
```

Comment:
We should add a warning here that filling tests is in some cases possible even on a faulty client (cases where we cannot do a post-check in the test, such as in some of the gas tests), which means the filling could succeed and an invalid test fixture could be created, so a client filling tests should not be considered as free of bugs.

Reply:
Just added the top-level filler page overview including DIRE WARNINGS about relying only on `fill` for correctness.

Reply:
It's here:
https://ethereum.github.io/execution-spec-tests/main/filling_tests/